pages/common/piactl.md

```diff
@@ -0,0 +1,36 @@
+# piactl
+
+> The command line tool for Private Internet Access.
```

Comment:
This could be misinterpreted. What about this instead?
```suggestion
> The command line tool for Private Internet Access, a commercial VPN provider.
```

Reply:
Very true. 👍🏻